osu.Game/Overlays/Settings/Sections/UserInterface/MainMenuSettings.cs

```diff
@@ -43,5 +52,17 @@ private void load(OsuConfigManager config)
                 }
             };
         }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            backgroundSourceDropdown.Current.BindValueChanged(source =>
+            {
+                const string not_supporter_note = "Changes to this setting will only apply with an active osu!supporter tag.";
+
+                backgroundSourceDropdown.NoteText = user.Value?.IsSupporter == false ? not_supporter_note : string.Empty;
```

Comment:
Should actually probably be

```suggestion
                backgroundSourceDropdown.NoteText = user.Value?.IsSupporter != true ? not_supporter_note : string.Empty;
```

? Not sure when this is permitted to be `null`, really...